### PR TITLE
Makefile: Add a way to build all Dockerfile. Fixes #2464

### LINKED ISSUE #2464: Makefile: Add a way to build all Dockerfile
As a follow-up to https://github.com/keptn/keptn/pull/2418:

Our makefile should introduce a command to build all Dockerfiles within the repo.

Our Dockerfiles live in sub-directories:

```
./helm-service/Dockerfile
./platform-support/openshift-route-service/Dockerfile
./mongodb-datastore/Dockerfile
./remediation-service/Dockerfile
./lighthouse-service/Dockerfile
./api/Dockerfile
./eventbroker/Dockerfile
./distributor/Dockerfile
./gatekeeper-service/Dockerfile
./shipyard-service/Dockerfile
./bridge/Dockerfile
./shipyard-controller/Dockerfile
./jmeter-service/Dockerfile
./upgrader/upgrade-061-062/Dockerfile
./upgrader/upgrade-062-070/Dockerfile
./upgrader/upgrade-060-061/Dockerfile
./configuration-service/Dockerfile
```

Please apply the following rules:
* Images are named after the parent-directory of the Dockerfile, e.g., when building `./platform-support/openshift-route-service/Dockerfile` the image should be name `${docker-org-name}/openshift-route-service`
  * The exception to that rules are:
    * `bridge` should be named `${docker-org-name}bridge2` (sorry for that, we'll clean that up at some point)
    * anything in the `upgrader` folder should be left out on purpose (it will be removed in the future)
* Images should be tagged using the following command: `git describe --abbrev=1 --tags` (or if you come up with something better, that's fine)

# Definition of Done

- [ ] Introduce a new variable called `DOCKER_ORG_NAME` (or something similar, if that makes more sense) which defaults to `keptn`
- [ ] Introduce a new command to the makefile, e.g., `make build-docker` according to the rules above
- [ ] If any error happens, the script should stop and print the error

### PR DESCRIPTION
**Pull request by Amab**
Fixes #2464

The building of all docker images can be launched using the `make build-docker` command.

### CODE REVIEW

**Comment by christian-kreuzberger-dtx** [comment]
Thanks! Could you please write at Issue https://github.com/keptn/keptn/issues/2464 that you're taking this task, so I can assign it to you?

File: make-scripts/build/build-docker.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+DOCKER_ORG_NAME="kept"
```
Comment (christian-kreuzberger-dtx) [inline make-scripts/build/build-docker.sh:5]: `DOCKER_ORG_NAME` should be set to `keptn`

File: make-scripts/build/build-docker.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  if [[ -v OVERRIDES[${DOCKER_DIR}] ]]; then
+    DOCKER_NAME="${OVERRIDES[${DOCKER_DIR}]}"
+    echo "Overridden docker tag $DOCKER_DIR -> $DOCKER_NAME"
+  else
+    DOCKER_NAME=$DOCKER_DIR
+  fi
+
+  DOCKER_TAG="$DOCKER_ORG_NAME/$DOCKER_NAME:$DOCKER_VERSION"
+
+  docker build -t $DOCKER_TAG "$DOCKER_PATH"
```
Comment (christian-kreuzberger-dtx) [inline make-scripts/build/build-docker.sh:52]: Please add some echo statements telling the user that we are building "Dockerfile" in directory etc...

Comment (Amab) [reply make-scripts/build/build-docker.sh:56]: Done.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
Thanks, LGTM.

Feel free to quickly present your PR in our [community meeting](https://github.com/keptn/community#%EF%B8%8F-community-meetings).

**Pull request merged**